File: sdkwork-ai-sdk-typescript/src/types/tool-call.ts
import type { FunctionCall } from './function-call';

/** 工具调用 */
export interface ToolCall {
  /** 工具调用ID */
  id?: string;
  /** 工具类型 */
  type?: string;
  /** 函数调用 */
  function?: FunctionCall;
}

File: sdkwork-ai-sdk-csharp/Models/ToolCall.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Ai.Models
{
    public class ToolCall
    {
        public string? Id { get; set; }
        public string? Type { get; set; }
        public FunctionCall? Function { get; set; }
    }
}
